cicd/gitops/monitoring-argo-cd-custom-resource-workloads.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="monitoring-argo-cd-custom-resource-workloads"]
+= Monitoring Argo CD custom resource workloads.adoc
+include::_attributes/common-attributes.adoc[]
+:context: monitoring-argo-cd-custom-resource-workloads
+
+toc::[]
+
+[role="_abstract"]
+With {gitops-title}, you can monitor the availability of Argo CD custom resource workloads for specific Argo CD instances. By monitoring Argo CD custom resource workloads, you have the latest information about the state of your Argo CD instances by enabling alerts for them. The Operator then triggers the alerts when the component workload pods such as application-controller, repo-server, or server of the correpsonding Argo CD instance are unable to come up for certain reasons and there is a drift between the number of ready replicas and the number of desired replicas for a certain period of time.
+
+You can enable and disable the setting for monitoring Argo CD custom resource workloads. 
+
+.Prerequisites
+
+* {gitops-title} is installed in your cluster.
+* Logged into Argo CD instance.
+* Prometheus is configured in your cluster and the Argo CD instance is in a namespace that you can monitor through Prometheus.
+* The `kube-state-metrics` service running in your cluster.
```

Comment:
@Srivaralakshmi can we add another prerequisite here:

If monitoring is being enabled in an Argo CD instance in a namespace that is _not_ watched by the default OpenShift monitoring stack (i.e any namespace that does not start with "openshift-*") then userworkload monitoring needs to be enabled on the cluster, so that the monitoring stack can pick up the created prometheusRule

Reply:
@jaideepr97  I have addressed your suggestion. PTAL. If all is ok, then pls provide LGTM. Thnx!